### PR TITLE
ORV2-878 Document management System (DMS) - Db changes

### PR DESCRIPTION
**Pull request by praju-aot**
# Description

Please provide a summary of the change and the issue fixed. Please include relevant context. List dependency changes.

This PR merges the DB changes required for DMS. It creates a new Schema called dms in ORBC DB and a new table called ORBC_DOCUMENT.

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- npm run test
- npm run test:e2e
- docker - to test the scripts are being executed as desired

## Checklist

- [x] I have read the [CONTRIBUTING](CONTRIBUTING.md) doc
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have already been accepted and merged

## Further comments

---

Thanks for the PR!

Any successful deployments (not always required) will be available below.
[Backend - Vehicles](https://onroutebc-436-backend-vehicles.apps.silver.devops.gov.bc.ca/) available
[Frontend](https://onroutebc-436-frontend.apps.silver.devops.gov.bc.ca/) available

Once merged, code will be promoted and handed off to following workflow run.
[Main Merge Workflow](https://github.com/bcgov/onroutebc/actions/workflows/merge-main.yml)

### CODE REVIEW

File: database/mssql/scripts/versions/revert/v_6_ddl_revert.sql [hunk 1]
```diff
@@ -0,0 +1,16 @@
+SET ANSI_NULLS ON
+GO
+SET QUOTED_IDENTIFIER ON
+GO
+SET NOCOUNT ON
+GO
+BEGIN TRANSACTION
+
+DROP TABLE [dms].[ORBC_DOCUMENT]
+
```
Comment (john-fletcher-aot) [inline database/mssql/scripts/versions/revert/v_6_ddl_revert.sql:12]: Need to drop [dms] schema after dropping the document table as well

File: database/mssql/test/versions/v_6_test.sh [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+# Retrieve arguments
+source ${SCRIPT_DIR}/utility/getopt.sh
+USAGE="-u USER -p PASS -s SERVER -d DATABASE"
+parse_options "${USAGE}" ${@}
+
+# All database tests for database version 5 are run from this shell script.
+# TESTS_DIR variable set by the calling test-runner script.
+
+# Test 5.1 - verify that the pdf template table exists
+TEST_5_1_RESULT=$(/opt/mssql-tools/bin/sqlcmd -U ${USER} -P "${PASS}" -S ${SERVER} -v DB_NAME=${DATABASE} -h -1 -i ${TESTS_DIR}/v_6_1_test.sql)
```
Comment (john-fletcher-aot) [inline database/mssql/test/versions/v_6_test.sh]: Should be TEST_6_1_RESULT (and below as well)

**Review by john-fletcher-aot — CHANGES_REQUESTED** [review]
Just a couple of minor changes

File: database/mssql/scripts/versions/revert/v_6_ddl_revert.sql (hunk 1 above)
Comment (praju-aot) [reply database/mssql/scripts/versions/revert/v_6_ddl_revert.sql:12]: Resolved

File: database/mssql/test/versions/v_6_test.sh (hunk 2 above)
Comment (praju-aot) [reply database/mssql/test/versions/v_6_test.sh]: Resolved

**Review by john-fletcher-aot — APPROVED** [review]

**Pull request merged**